meetings/2023-02-01.md

```diff
@@ -50,36 +57,125 @@ URL: https://blog.gitter.im/2023/01/16/gitter-is-going-fully-native-matrix-in-fe
+
+* WT: I wanted to confirm that I think this is not a priority for CG/WG. If we have a chat that works, it works, and if it's to be a showcase of Solid, which I would support, then I think it can't just be one central chat client over one server storing all the data. we should have multiple clients by multiple developers with data storage in user's pods for it to be showcased at all. 
+* PAC: Regarding the distribution of the data, I understand it is important to demonstrate this in a Solid showcase app, but it raises the question of archives, because I expect personal pods might come and go, and do we accept the drawback of losing a part of the conversation? There's a tradeoff. 
+* TBL: If you look at IRC chats on W3C there are RDF descriptions of IRC chats going back for 25 years. When solid chat runs it just creates the archive ??? . As people chat it adds messages into the archive directly. So with chat systems there is the question of how to archive. With Solid Chat the archive is the chat. At the moment solidproject.org has a persistence policy for the TR space and we can make a persistence policy. Once you write a chat into Solid chat, that URL of that message will be available forever. 
+* SC: Is there an authoritative version of a message? Is it one storage that has that? 
+* TBL: At the moment a pod has it, yes.
+* TBL: You can keep synchronized copies of that chat. 
+* SC: Maybe it needs to be on solidproject.org or solidcommunity.net
+* eP: When it comes to data distribution I agree on a chat room on a single storage. At the same time I think each user should be able to create chats within their storage. Replication would be an extra, nice to have. 
+* TBL: with solidOs you can create a folder in your pod and put all kinds of things including a chat and that use access control systems to give groups access to them. When you look at someone's profile at the moment there is a "chat with me" button that creates a chat between the two of you.
+* WT: I don't think it is a problem to store a chatroom in a single pod, but we cannot depend on this choice of one ??? a chat based on Solid would be a perfect choice that we will encounter limits on interoperability as soon as we try to chat implementations to work together. It is not the case one follows the other in the way they store data.
+* TBL: The way we're working at the moment is that we are developing things ??? so the chat shapes, when the Oviedo students had as a student project to make a chat, they were given the chat shapes so that the requirement was that it will be compatible. So when Jackson made a chat, that is compatible I think. What the Solid CG could look at doing is making those specs match. Taking those existing shapes we have putting them into some lightweight document that other people can see. we can argue/have discussions about whether ??? and whether you prefer a different ontology.. I hope that stuff gets standardized soon. 
```

Comment:
```suggestion
* TBL: The way we're working at the moment is that we are developing things ??? like the chat shapes. When the Oviedo students had as a student project to make a chat, they were given the chat shapes, so the requirement was that it would be compatible. So when Jackson made a chat, that is compatible, I think. What the Solid CG could look at doing is making those specs match. Taking those existing shapes we have, and putting them into some lightweight document that other people can see. we can argue/discuss about whether ??? and whether you prefer a different ontology. I hope that stuff gets standardized soon. 
```